services/web/server/src/simcore_service_webserver/director_v2_handlers.py

```diff
@@ -26,42 +31,65 @@
 @permission_required("project.read")
 @log_decorator(logger=log)
 async def start_pipeline(request: web.Request) -> web.Response:
-    director2_settings: Directorv2Settings = get_settings(request.app)
+    client = DirectorV2ApiClient(request.app)
+
+    run_policy = get_run_policy(request.app)
+    assert run_policy  # nosec
+
+    user_id = UserID(request[RQT_USERID_KEY])
+    project_id = UUID(request.match_info["project_id"])
```

Comment:
```suggestion
    project_id = ProjectID(request.match_info["project_id"])
```
Or that does not work?